src/app.ts

```diff
@@ -19,17 +19,18 @@ import { ByteStream } from '@zondax/ledger-js/dist/byteStream'
 
 import { P1_VALUES, PUBKEYLEN } from './consts'
 import { Account, AccountType, ResponseAddress, VaultAccount } from './types'
-import { ResponseSign } from './types'
+import { MsgSigner, ResponseSign } from './types'
 
 export class SpaceMeshApp extends BaseApp {
     static _INS = {
         GET_VERSION: 0x00 as number,
         GET_ADDR: 0x01 as number,
         SIGN: 0x02 as number,
+        SIGN_MESSAGE: 0x03 as number,
```

Comment:
This won't be backward compatible, as we are changing meaning to current values. Isn't that a problem?

Reply:
I fixed to keep backward compatible